tests/models/test_llava_next.py

```diff
@@ -14,21 +18,21 @@
     "The assistant gives helpful, detailed, and polite answers to the human's "
     "questions.")
 
-# The image token is placed before "user" on purpose so that the test can pass
 HF_IMAGE_PROMPTS = IMAGE_ASSETS.prompts({
     "stop_sign":
-    f"{_PREFACE} <image>\nUSER: What's the content of the image?\nASSISTANT:",
+    f"{_PREFACE} USER: <image>\nWhat's the content of the image? ASSISTANT:",
     "cherry_blossom":
-    f"{_PREFACE} <image>\nUSER: What is the season?\nASSISTANT:",
+    f"{_PREFACE} USER: <image>\nWhat is the season? ASSISTANT:",
 })
 
 
 def iter_llava_next_configs(model_name: str):
+    # Need to use the max possible feature size for profile_run
     image_hw_to_feature_size = {
-        (336, 336): 1176,
+        (336, 336): 2928,
         (672, 672): 2928,
-        (1344, 336): 1944,
-        (336, 1344): 1890,
+        (1344, 336): 2928,
```

Comment:
why 2928? How's it calculated?

Reply:
It's mostly a holdover from the previous code. This is not used anymore but we still need to pass a value to the config until your PR removes it.